src/Temporalio.Extensions.Hosting/TemporalWorkerServiceId.cs

```diff
@@ -0,0 +1,12 @@
+namespace Temporalio.Extensions.Hosting
+{
+    /// <summary>
+    /// Represents the unique identifier for a hosted Temporal Worker service.
+    /// </summary>
+    public record TemporalWorkerServiceId(
+        string TaskQueue,
+        string? Version,
+        bool IsBuildId)
+    {
+    }
```

Comment:
```suggestion
        bool IsBuildId);
```

I think this should work, unsure